acquia_lift_profiles/acquia_lift_profiles.module

```diff
@@ -147,27 +147,35 @@ function acquia_lift_profiles_page_build(&$page) {
   }
   $js_path = variable_get('acquia_lift_profiles_js_path', '');
   $account_name = variable_get('acquia_lift_profiles_account_name', '');
-  $udf_mappings = variable_get('acquia_lift_profiles_udf_mappings', array());
+  $capture_identity = variable_get('acquia_lift_profiles_capture_identity', FALSE);
+  $mappings = variable_get('acquia_lift_profiles_udf_mappings', array());
+  $acquia_lift_profiles_vocabulary_mappings = variable_get('acquia_lift_profiles_vocabulary_mappings', array());
+  if (!empty($acquia_lift_profiles_vocabulary_mappings)) {
+    $mappings['vocabulary'] = $acquia_lift_profiles_vocabulary_mappings;
```

Comment:
This setup totally works haha. I started off by fully coding a separate data flow for vocabulary mappings mimicking udf mappings. Once finished, I started to reduce the duplicated code backwards. It turns out I can completely use udf mappings code, as long as I put all vocabulary mappings values as another set of udfValues alongside the 'person', 'touch', and 'event'.

Yes qunit and simpletests are coming up.